src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
-'use strict';
+'use strict'
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+promise1.then((result) => {
+  const message = document.createElement('div');
+
+  message.classList.add('message');
+  message.textContent = result;
+  document.body.appendChild(message);
+
+  const promise2 = new Promise((resolve, reject) => {
+    setTimeout(() => {
+      resolve('Promise was rejected!');
```

Comment:
The provided text should be 'Promise was rejected!', but you have resolved the promise with this text. Please change `resolve` to `reject` to correctly reject the promise. 🚀